fiaisis/frontend
# Remove all references to demo_plugin and fix icon

Closes #16, closes #72.

## How to test

Update SciGateway's `settings.json` to point to "fia" instead of "demo_plugin". After running `yarn build` on the frontend, run the plugin through SciGateway. Should work.
